docs/rp2/quickref.rst

```diff
@@ -202,13 +234,16 @@ See :ref:`machine.RTC <machine.RTC>` ::
     from machine import RTC
 
     rtc = RTC()
-    rtc.datetime((2017, 8, 23, 2, 12, 48, 0, 0)) # set a specific date and time
+    rtc.datetime((2017, 8, 23, 1, 12, 48, 0, 0)) # set a specific date and time
     rtc.datetime() # get date and time
 
 WDT (Watchdog timer)
 --------------------
 
-Is there a watchdog timer?
+The RP2XXX port has a watchdog which is a countdown timer that can restart parts
+of the chip if it reaches zero. This helps in restarting the processor if
+software gets stuck in an infinite loop. The programmer must periodically write
+a value to the watchdog to stop it from reaching zero.
```

Comment:
The user doesn't need to write a value to the watchdog, they just need to call the feed() method (which will probably write a register but that's an implementation detail that doesn't need to be explained here).